libs/ucdao/src/lib/ucdao-confirm-modal.tsx

```diff
@@ -39,17 +41,25 @@ export function ConfirmModal({
 
         <div className="flex w-full flex-col">
           <div className="pb-[24px] pt-[24px] sm:pt-[4px]">
-            <ModalHeading>Select wallet</ModalHeading>
+            <ModalHeading>
+              {t('select-wallet-heading', 'Select wallet', { ns: 'uc-dao' })}
```

Comment:
please add namespace to hook args. if in file used only one namespace
```suggestion
              {t('select-wallet-heading', 'Select wallet')}
```

Reply:
okay but here ConfirmModal uses two namespaces

will do this in DaoBalanceConfirmAmount